_posts/2017-01-07-ubuntuinstall.md

```diff
@@ -9,7 +9,7 @@ hidden: true
 
 ## Ubuntu installation guide
 
-* Insert the bootable USB stick, reboot the system and instruct the **BIOS** to boot from the USB by pressing a special function key (usually **F9**, **F10** or **F12** depending on your device) after the boot up a new grub screen should appear on the screen. From the menu select **Install Ubuntu** and press **Enter**.
+* Insert the bootable USB stick, reboot the system and instruct the **BIOS** to boot from the USB by pressing a special function key (usually **F1**, **F2** **F9**, **F10**, **F12** or maybe **del** depending on your device) after the boot up a new grub screen should appear on the screen. From the menu select **Install Ubuntu** and press **Enter**.
```

Comment:
(similar to linux mint, advanced startup worked on all the laptops we tackled this linux install fest)